src/platforms/java/migration.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Migration Guide
+sidebar_order: 1000
+---
```

Comment:
Adding this to the frontmatter ensure that there's a description for <PageGrid /> lists
```suggestion
description: "Learn about migrating from io.sentry:sentry 1.x t io.sentry:sentry 3.x."
---
```